src/models/Bookmark.ts

```diff
@@ -0,0 +1,22 @@
+import { ObjectId } from "mongodb";
+import mongoose, { model, Schema, Document } from "mongoose";
+
+export interface Bookmark extends Document {
+  tweetId: ObjectId;
+  ownerId: ObjectId;
+}
+
+const BookmarkSchema: Schema = new Schema<Bookmark>({
+  tweetId: {
+    type: mongoose.Schema.Types.ObjectId,
```

Comment:
why did you use type "type: mongoose.Schema.Types.ObjectId", instead "type: ObjectId"?